src/status_im/ui/screens/chat/views.cljs

```diff
@@ -350,24 +369,11 @@
     (re-frame/dispatch [:navigate-back])))
 
 (defn topbar []
```

Comment:
if we want to have two version better to rename old one with _old , `topbar_old` overwise it will be a mess with different names for different versions